### PR TITLE
Implement app header

### PR DESCRIPTION
**Pull request by dmrib**
Depends on #1 .

First commit is: `Add general app styling sheet`

### CODE REVIEW

**Review by HeitorBoschirolli — COMMENTED** [review]
There will be no version update commit?

File: linguicator-editor/src/App.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
-import { Typography } from 'antd';
 import React from 'react';
+import Header from './components/Header/Header';
+import './App.css'
+import 'antd/dist/antd.css';
 
 
 function App() {
```
Comment (HeitorBoschirolli) [inline linguicator-editor/src/App.js]: What do you think about adding a comment describing each component describing what will be rendered?

**Review by HeitorBoschirolli — CHANGES_REQUESTED** [review]
Just one more thing

File: linguicator-editor/src/App.js (hunk 1 above)
Comment (dmrib) [reply linguicator-editor/src/App.js]: Done!

**Review by HeitorBoschirolli — COMMENTED** [review]
What do you think about renaming `src/components/Header/Header.js` to `src/components/Header/index.js`?

**Review by HeitorBoschirolli — APPROVED** [review]
+1

**Pull request merged**